students/sangwoong/users/views.py

```diff
@@ -1,3 +1,49 @@
 from django.shortcuts import render
 
 # Create your views here.
+import json, re
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignUpView(View):
+  
+  def post(self, request) :
+   try:
```

Comment:
1. 불필요한 공백은 제거해 주세요.
2. python에서 권장하고 있는 들여쓰기의 간격은 4칸입니다. 현재 2칸씩으로 되어 있으니 전부 수정해 주세요.

```suggestion
class SignUpView(View):
    def post(self, request) :
         try:
```